spec/models/levelling_up_premium_payments/dqt_record_spec.rb

```diff
@@ -18,21 +18,57 @@
   let(:record) do
     OpenStruct.new(
       {
-        degree_codes: [],
+        degree_codes: degree_codes,
         itt_subjects: ["Applied Mathematics"],
-        itt_subject_codes: ["G1100"],
+        itt_subject_codes: itt_subject_codes,
         itt_start_date: Date.parse("1/9/2018"),
         qts_award_date: Date.parse("31/8/2019"),
         qualification_name: "BA"
       }
     )
   end
 
+  let(:itt_subject_codes) { [] }
+  let(:degree_codes) { [] }
+
   describe "#eligible?" do
     subject(:eligible?) { dqt_record.eligible? }
 
-    it "returns true" do
-      expect(subject).to be true
+    context "without ITT and degree codes" do
+      it { is_expected.to be false }
```

Comment:
I prefer `it { is_expected.to be_eligible }` but I don't mind

Reply:
Done in a1c5cb654b5e6474e9fc2bd2d0cd21473c741fc7